### DISCUSSION TITLE
Biometric (webauthn) login

### DISCUSSION THREAD

**Discussion by KTibow** [home-assistant/frontend#7914] [Other feature requests] [answered]
[Webauthn](https://webauthn.io/) lets you log in using private and public keys. It would be nice to have this supported, so you can log in with your face or fingerprint or security key instead of a password.

**Comment by frenck** [comment, marked as answer]
This is not a frontend related request, the Home Assistant backend needs to support it; thus the right location for this is currently the community forum

**Comment by bramkragten** [comment]
https://github.com/home-assistant/core/pull/19484
